uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-assignment.controller.js

```diff
@@ -242,6 +246,29 @@ define([
       });
     }
 
+    /**
+     * Opens the contact's relationship creation modal. When a relationship is
+     * successfully created it reloads the default assignees for each activity.
+     */
+    function createRelationshipForTargetContact () {
+      var formUrl = CRM.url('civicrm/contact/view/rel', {
+        action: 'add',
+        cid: vm.assignment.client_id
+      });
+
+      CRM.loadForm(formUrl)
```

Comment:
We have a service in L&A Reqangular for that, we may want to have the same service in T&A?

cc @AkA84

Reply:
I could use the `loadForm` from reqangular, but there are some issues:

* I also need `CRM.url` which is not in reqangular.
* The reqangular one also suffers from the digest issue, so it needs to be augmented.

In short, there's no clear benefit on using the reqangular wrapper, the CRM variable can be accessed without issues from tests and all of its services can be used directly without having to wrap them and add more tests on top of them.

What do you think?

Reply:
I think leave it as is.